presidio-analyzer/presidio_analyzer/recognizer_registry/recognizer_registry_provider.py

```diff
@@ -0,0 +1,333 @@
+import re
```

Comment:
Moved this file (and the recognizer_registry) into their own folder. Not sure why Github shows this as a new file

Reply:
Love it